README.md

```diff
@@ -44,6 +44,7 @@ A dbt profile can be configured to run against Trino using the following configu
 | method                         | The Trino authentication method to use                                                                       | Optional (default is `none`, supported methods are `ldap`, `kerberos`, `jwt`, `oauth` or `certificate`) | `none` or `kerberos`             |
 | user                           | Username for authentication                                                                                  | Optional (required if `method` is `none`, `ldap` or `kerberos`)                                         | `commander`                      |
 | password                       | Password for authentication                                                                                  | Optional (required if `method` is `ldap`)                                                               | `none` or `abc123`               |
+| impersonate_user               | impersonate user, as X-Trino-User                                                                            | Optional (applicable if `ldap`)                                                                         | `impersonated_tom`               |
```

Comment:
Description: Username override, used for impersonation.

Reply:
Parameter name: `impersonation_user`

Reply:
Updated :)